test/unitTest/java/org/thoughtcrime/securesms/util/SwipeGesturedetectorUnitTest.java

```diff
@@ -0,0 +1,61 @@
+package org.thoughtcrime.securesms.util;
+
+import android.view.MotionEvent;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import org.thoughtcrime.securesms.ConversationActivity;
+
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.powermock.api.mockito.PowerMockito.when;
+
+/**
+ * Created by Claudia on 2018-02-09.
+ */
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({MotionEvent.class, ConversationActivity.class, SwipeGestureDetector.class})
+public class SwipeGesturedetectorUnitTest {
+
+    private ConversationActivity mockConversationActivity;
+    private SwipeGestureDetector testSwiper;
+    private MotionEvent motionEvent1;
+    private MotionEvent motionEvent2;
+
+    @Before
+    public void setUp() throws Exception {
```

Comment:
Is it the GestureDetector that throws an Exception? Why is the method marked as `throws Exception`?